lib/circuit.js

```diff
@@ -557,22 +584,27 @@ class CircuitBreaker extends EventEmitter {
     }
     const args = Array.prototype.slice.call(rest);
 
+    const cacheKey = this.options.cacheGetKey.apply(this, rest);
```

Comment:
Done in [c2e54b8](https://github.com/nodeshift/opossum/pull/739/commits/c2e54b8f4b6b9e3fddf22d0ce46b610884151eff). Still need to have variable, because it used lower in storing value in cache.